apisix/cli/ops.lua

```diff
@@ -269,11 +269,21 @@ Please modify "admin_key" in conf/config.yaml .
                  "your openresty, please check it out.\n")
     end
 
+    --- http is enabled by default
     local enable_http = true
-    if not yaml_conf.apisix.enable_admin and yaml_conf.apisix.stream_proxy and
-        yaml_conf.apisix.stream_proxy.only ~= false
-    then
-        enable_http = false
+    --- stream is disabled by default
+    local enable_stream = false
+    local http = "http"
+    local stream = "stream"
+    if yaml_conf.apisix.proxy_mode then
+        -- check for "http" as prefix 
+        if string.sub(yaml_conf.apisix.proxy_mode,1,#http) ~= http then
```

Comment:
Literally compare with "http", "stream" and "http&stream" here.

Reply:
done